.circleci/config.yml

```diff
@@ -121,6 +121,27 @@ jobs:
           name: Reporting code coverage...
           command: bash <(curl -s https://codecov.io/bash) -F << parameters.package >> -X gcov
 
+  lint:
+    resource_class: medium+
+    docker:
+      - image: cimg/node:16.13
+    environment:
+      TRACING_SERVICE_NAME: ci-lint
+      TRACING_CONSOLE_EXPORTER_ENABLED: true
+    steps:
+      - base-install
+      - run:
+          name: Linting
+          command: |
+            PACKAGES=(\
+            'fxa-shared' \
+            'fxa-auth-server' \
+            )
+            for p in "${PACKAGES[@]}"; do
+              (cd packages/$p && yarn lint)
+            done
+      - jira/notify
+
```

Comment:
I see that this workflow runs in parallel with our other workflows, so at the end of the day, it's not increasing the wall clock time of our CI overall, but it did take nearly 3 minutes to run (with more than 90% of that time setting up and tearing down). Compare this to less than 6 seconds for `yarn lint` to run locally (for me anyway).

What I'm trying to say is: I'm wondering if it makes sense to only add this to the Husky pre-commit hook and leave it out of CI? Maybe it's worth checking in with @clouserw or others on the team to see what they think (e.g. asking in #fxa-team)?

Reply:
> I'm wondering if it makes sense to only add this to the Husky pre-commit hook

This is how we get into the same situation we're in now - if someone doesn't use husky locally we'll have lint errors all over the place. We need CI acting as the source of truth here, and preventing us from merging code that violates our code standards.

In other words, this is definitely worth our CI time.